src/stylesheets/components/_key-factors.scss

```diff
@@ -0,0 +1,291 @@
+/*----------- Key Factors -----------*/
+
+.key-factors {
+  color: $florence-gray;
+  margin-top: 150px;
+
+  @media ($screen-sm) {
+    margin-top: 65px;
+  }
+
+  h3 {
+    color: $florence-orange;
+    font-size: 1em;
+    margin-bottom: -8px;
+    text-transform: uppercase;
+  }
+}
+
+.section-title {
+  color: $florence-blue;
+  font-size: 0.65em;
+  letter-spacing: 2px;
+  text-align: center;
+  text-transform: uppercase;
+  width: 100%;
+}
+
+.section-intro {
+  color: $florence-blue;
+  font-family: $secondary-font;
+  font-size: 1.7em;
+  line-height: 1.6em;
+  margin: auto;
+  text-align: center;
+  width: 57vw;
+
+  @media ($screen-sm) {
+    font-size: 1.4em;
+    width: 85vw;
+  }
+}
+
+.factor-bg {
+  background-image: url('../images/body-burst.png');
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: 1000px auto;
```

Comment:
Should be able to combine all these background properties into the [shorthand version](https://developer.mozilla.org/en-US/docs/Web/CSS/background).